### DISCUSSION TITLE
Why not using SubscribeWithContext in example subscribe.go

### DISCUSSION THREAD

**Discussion by yinjilong** [gopcua/opcua#581] [General]
Most functions use `WithContext` method, `sub.Cancel` also uses context parameter,
I wonder if here `SubscribeWithContext`  should be used or not.
Thanks

**Comment by magiconair** [comment]
The `XXXWithContext` methods will replace the ones without in v0.5 (#553). You should use the `WithContext` methods now. Did we miss one in an example?

**Comment by magiconair** [comment]
Yes, we did. Fixed it. Thank you.

**Reply by yinjilong** [reply]
Thanks for your kindness
